src/test/java/com/wikia/webdriver/testcases/mercurytests/SEOTests.java

```diff
@@ -142,4 +150,64 @@ public void MercurySEOTest_001_MetaTags_CanonicalLink() {
         seo.isOgTypeArticle()
     );*/
   }
+
+  @Test(groups = {"MercurySEOTest_002", "MercurySEOTests", "Mercury"})
+  public void MercurySEOTest_002_MetaTags_Robots_SectionToMainPage() {
+    SEOUtils seoUtils = new SEOUtils(driver);
+    wikiURL = urlBuilder.getUrlForWiki(MercuryWikis.MERCURY_CC);
+    CuratedContentPageObject section = new CuratedContentPageObject(driver);
+    section.openCuratedContentPage(wikiURL, MercurySubpages.CC_SECTION_CATEGORIES);
+
+    Assertion.assertTrue(seoUtils.isRobotsMetaTagSet(), "Robot Meta Tags are not set when supposed to");
+    Assertion.assertTrue(seoUtils.isAttributesListPresentInRobotsMetaTag(
+        ROBOTS_TAG_ATTRIBUTES_NOINDEX_FOLLOW), "Robot Meta Tags are different than expected");
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "Robot Meta Tags are different than expected" 4 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1192)